complete/src/main/java/price/checker/domain/StockPrice.java

```diff
@@ -0,0 +1,125 @@
+package price.checker.domain;
+
+import javax.persistence.*;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * This is the model/schema for the StockPrice table stored in MySQL
+ * It uses an ID class called "StockPriceId"
+ * Because the StockPrice table uses a composite key, it's required for a
+ * separate, serializable class to be created for the key
+ */
+@Entity //This tells Hibernate to make a table out of this class
+@Table(name="Stock_Price")
+@IdClass(StockPriceId.class) //we need this and have the key in
+                            // another class because it is a composite key
+public class StockPrice {
+
+    //The symbol of the company and the date is the composite key of the table
+    @Id
+    @Column(length = 5)
+    private String symbol;
+
+    @Id
+    @Column

```

Comment:
It seems like you don't need @Column unless you are setting specific properties related to that column (e.g. length or name) @Entity automatically will assume every field in the class should be a column